ee/clickhouse/queries/test/test_trends.py

```diff
@@ -852,3 +853,9 @@ def test_filtering_with_action_props(self):
         )
 
         self.assertEqual(response[0]["count"], 2)
+
+    def test_trends_math_without_math_property(self):
```

Comment:
I would ideally have a test case on the API level as you want to test here that the proper response is sent back.

Reply:
Fair point. Trends don't have any api-level tests though, so let's leave it here for now. :)

Reply:
They do: https://github.com/PostHog/posthog/blob/master/posthog/api/test/test_insight.py#L249

And you can add more CH specific ones here: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/views/test/test_clickhouse_insights.py#L23